Makefile

```diff
@@ -5,20 +5,17 @@
 default: test
 
 test:
-	python -m unittest discover
-
-coverage:
-	coverage run -m unittest discover
-	coverage report -m
+	tox -e py27,py34
 
 pylint:
 	pylint edx_lint test setup.py
 
 clean:
 	-rm -rf *.egg-info
-	-rm -f *.pyc */*.pyc */*/*.pyc */*/*/*.pyc */*/*/*/*.pyc */*/*/*/*/*.pyc
-	-rm -rf __pycache__ */__pycache__ */*/__pycache__ */*/*/__pycache__ */*/*/*/__pycache__ */*/*/*/*/__pycache__
+	-find . -name '*.pyc' -delete
+	-find . -name '__pycache__' -delete
```

Comment:
I was looking into this Friday morning. This will fail if the directory is not empty.